From 9d24257a5ca00e6de0675d0c2dbd3d824f23ebf9 Mon Sep 17 00:00:00 2001
From: Edward Hervey <edward@centricular.com>
Date: Wed, 21 Feb 2018 08:49:15 +0100
Subject: [PATCH] stun: Also rename windows-specific function

Like all other instances of nice_RAND_bytes that were renamed
to nice_RAND_nonce.

Fixes the windows build
---
 stun/rand.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stun/rand.c b/stun/rand.c
index 3ac6741..25dc402 100644
--- a/stun/rand.c
+++ b/stun/rand.c
@@ -45,7 +45,7 @@
 #include <windows.h>
 #include <wincrypt.h>
 
-void nice_RAND_bytes (uint8_t *dst, int len)
+void nice_RAND_nonce (uint8_t *dst, int len)
 {
   HCRYPTPROV hCryptProv;
   LPCSTR container = "Libnice key container";
-- 
2.14.3

